packages/puppeteer-core/src/api/locators/locators.ts

```diff
@@ -113,6 +114,29 @@ export abstract class Locator<T> extends EventEmitter<LocatorEvents> {
     return RaceLocator.create(locators);
   }
 
+  /**
+   * Waits for multiple locators to locate elements in DOM and
+   * calls the action on the last locator waiting for action-specific preconditions.
+   * If any of the elements identified by locators is not found, the last locator
+   * times out.
+   * @example
+   *
+   * ```ts
+   * await Locator.last([
```

Comment:
is there an example where one would want to do an action other than wait using Locator.last? it seems like the Last locator is only useful with waiting?

Reply:
In some scenarios, users may be sure that some certain elements may appear later than other elements and also the last element depends on the earlier ones to do the operation. I think wait is more often used. Do I also need to add an operation example?

Reply:
I think if you need to wait multiple locators for majority of the cases:

`await Promise.all([page.locator('#test1'), page.locator('#test2')].map(l => l.wait()))` is more easy to use and more clear what it does. Is there a good scenario when one wants to click or do another action on the last element that appears in DOM?

Reply:
I can't list a very specific case. I think in some scenarios web page is changing, we have to wait for the last element to appear and do the operation like type or click which depend on the previous elements. And the element appearance order is flexible.